src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const DAY_PRICE = 40;
+  const BASE_PRICE = days * DAY_PRICE;
```

Comment:
```suggestion
  const basePrice = days * DAY_PRICE;
```

Reply:
It's better to name it in camelCase notation because we count it based on the 'days' variable that could change and depend on function arguments

Reply:
done